setup.py

```diff
@@ -309,6 +309,7 @@ def do_setup():
         scripts=['airflow/bin/airflow'],
         install_requires=[
             'alembic>=0.9, <1.0',
+            'cached_property~=1.5',
```

Comment:
Is it an Apache approved library?

Reply:
BSD, which is Cat-A http://www.apache.org/legal/resolved.html#category-a